Lib/dataclasses.py

```diff
@@ -1549,11 +1550,20 @@ class C(Base):
         seen.add(name)
         annotations[name] = tp
 
+    def annotate_method(format):
+        if format != 1:
```

Comment:
You'll probably have to support the other formats too, and test that all three formats work as expected.

Reply:
Yes, sure. This is just a WIP.